.github/chronographer.yml

```diff
@@ -0,0 +1,11 @@
+---
```

Comment:
Add https://github.com/apps/psf-chronographer to make use of this config. It's a simple check for whether the PR authors didn't forget to add a change note. The config is currently set up to ignore PRs from known bots.